src/commonMethods/commonFunctions.js

```diff
@@ -174,7 +210,6 @@ export const screenshot = async function() {
   let url = `http://${constants.host}:80/Service/Snapshot/Capture?${moment().valueOf()}`
   // create a new promise inside of the async function
   let bufferData = new Promise((resolve, reject) => {
-    //TODO : Replace _http by using this.$http helper
```

Comment:
Why was this todo comment removed?

Reply:
This was removed by mistake. Reverting this line to the original.